src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('.population')];
+const toNumbers = population.map((element) => {
+  return parseInt(element.textContent.replace(/,/g, ''));
+});
+
+const totalSum = toNumbers.reduce((sum, element) => sum + element, 0);
```

Comment:
It's a good practice to provide an initial value for the reduce method, even though in this case, it defaults to the first element of the array when not provided. It makes the code more explicit and helps avoid potential bugs in the future if the code changes.